Packs/PowershellRemoting/Integrations/PowerShellRemoting/PowerShellRemoting.Tests.ps1

```diff
@@ -0,0 +1,4 @@
+BeforeAll {	
+    . $PSScriptRoot\demistomock.ps1	
+    . $PSScriptRoot\PowerShellRemoting.ps1
```

Comment:
what does this do?

Reply:
This imports demistomock and the integration - but since we don't have any unit tests, it's not really doing anything.